client/src/pages/GraphiQL.js

```diff
@@ -13,7 +13,7 @@ class GraphiQL extends Page {
 	static propTypes = {...pagePropTypes}
 
 	constructor(props) {
-		super(props, PAGE_PROPS_NO_NAV, CLEAR_SEARCH_PROPS)
+		super(props, PAGE_PROPS_NO_NAV, DEFAULT_SEARCH_PROPS)
```

Comment:
Change this to simply `super(props, PAGE_PROPS_NO_NAV)` everywhere and ditch the import.